pkg/settings/registry.go

```diff
@@ -135,6 +188,12 @@ func register(class Class, key, desc string, s internalSetting) {
 	if _, ok := registry[key]; ok {
 		panic(fmt.Sprintf("setting already defined: %s", key))
 	}
+	if strings.Contains(key, "sql.defaults") {
+		if _, ok := sqlDefaultSettings[key]; !ok {
+			panic(fmt.Sprintf(
+				"new sql.defaults cluster settings: %s is not needed now that `ALTER ROLE..SET` syntax is supported", key))
```

Comment:
nit: new sql.defaults cluster setting detected: %s is not needed now that `ALTER ROLE ... SET` syntax is supported